test/sharness/t0170-dht.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+
+test_description="Test dht command"
+
+. lib/test-lib.sh
+
+# start iptb + wait for peering
+NUM_NODES=5
+test_expect_success 'init iptb' '
+  iptb init -n $NUM_NODES --bootstrap=none --port=0 &&
+  startup_cluster $NUM_NODES
+'
+
+test_expect_success 'peer ids' '
+  PEERID_0=$(ipfsi 0 id --format="<id>") &&
+  PEERID_2=$(ipfsi 2 id --format="<id>")
+'
+
+# ipfs dht findpeer <peerID>
+test_expect_success 'findpeer' '
+  ipfsi 1 dht findpeer $PEERID_0 >actual &&
+	egrep "/ip4/127.0.0.1/tcp/.*" actual >/dev/null &&
```

Comment:
test that the address has the correct port, we can get this info from running `ipfs id -f "<addrs>"` on the peer in question.